Examples/test-suite/javascript/abstract_access_runme.js

```diff
@@ -1,6 +1,8 @@
-var abstract_access = require("abstract_access");
+var abstract_access = require("./abstract_access");
```

Comment:
Why this change? Is it because of electron support?

Reply:
Yes, it seems in some context it would not find modules even though it is at the right place. Electron, as you guessed. Again, maybe there is a better solution but I figured it would be acceptable to at least start getting tests green on Travis.

Reply:
Yes, it seems in some context it would not find modules even though it is at the right place. Electron, as you guessed. Again, maybe there is a better solution but I figured it would be acceptable to at least start getting tests green on Travis.